src/views/notifications/Latest.vue

```diff
@@ -0,0 +1,324 @@
+                                </div>
+                              </div>
+                              <input class="form-check-switch ml-auto" type="checkbox" v-model="this.single_notification.seen">
+                            </div>
+                          </div>
+                          <div class="col-span-12 xxl:col-span-12">
+                            <hr class="my-5">
+                          </div>
+                          <div class="col-span-12 xxl:col-span-6 mb-4">
+                            <div>
+                              <label for="update-icon-notification" class="form-label">
+                                Icon
+                              </label>
+                              <input
+                                id="update-icon-notification"
+                                type="text"
+                                :class="'form-control' + (this.validation_error?.icon != null ? ' border-theme-6' : '')"
+                                placeholder="Enter Icon"
+                                v-model="this.single_notification.icon"
+                                :disabled="parseInt(this.single_notification.type) !== 1"
+                              />
+                              <div v-if="parseInt(this.single_notification.type) !== 1" class="text-theme-6 mt-2 mb-4">
+                                User Type is selected!
+                              </div>
+                              <div v-if="this.validation_error?.icon != null" class="text-theme-6 mt-2 mb-4">
+                                {{ this.validation_error?.type[0] }}
+                              </div>
+                            </div>
+                          </div>
+                          <div class="col-span-12 xxl:col-span-6 mb-4">
+                            <div>
+                              <label class="form-label">
+                                User
+                              </label>
+                              <TailSelect
+                                :class="'form-control' + (this.validation_error?.target_id != null ? ' border-theme-6' : '')"
+                                :options="{
+                                  search: true,
+                                  classNames: 'w-full'
+                                }"
+                              >
+                                <option :value="user.id" v-for="user in this.users" v-bind:key="user.id" :selected="this.single_notification.target_id === user.id">
+                                  {{ user.name }}
+                                </option>
+                              </TailSelect>
+                              <div v-if="parseInt(this.single_notification.type) !== 2" class="text-theme-6 mt-2 mb-4">
+                                Icon Type is selected!
+                              </div>
+                              <div v-if="this.validation_error?.target_id != null" class="text-theme-6 mt-2 mb-4">
```

Comment:
Unnecessary `!= null`

Reply:
Solved in https://github.com/KrbDevelopment/Articly-Frontend/commit/5172bc53bad0121ee21691def6813959202839fa